calabash/Classes/WebViewQuery/LPWebQuery.m

```diff
@@ -67,6 +67,8 @@ + (NSArray *) arrayByEvaluatingQuery:(NSString *) query
 
   for (NSDictionary *d in queryResult) {
     NSMutableDictionary *dres = [NSMutableDictionary dictionaryWithDictionary:d];
+
+    // Inexplicably, rect.x and rect.y are the _center_.
```

Comment:
I think this is an unexpected behavior.  They should be named rect.center_x and rect.center_y.